xml/System.Data.Spatial/DbGeography.xml

```diff
@@ -354,8 +354,8 @@
         <ReturnType>System.Nullable&lt;System.Double&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the elevation (Z coordinate) of this DbGeography value, if it represents a point. &lt;returns&gt;The elevation (Z coordinate) value of this geography value, if it represents a point; otherwise <see langword="null" />.&lt;/returns&gt;</summary>
-        <value>The elevation (Z coordinate) of this DbGeography value.</value>
+        <summary>Gets the elevation (Z coordinate) of this DbGeography value, if it represents a point.</summary>
+        <value>The elevation (Z coordinate) value of this geography value, if it represents a point; otherwise <see langword="null" />.</value>
```

Comment:
```suggestion
        <value>The elevation (Z coordinate) value of this geography value, if it represents a point; otherwise, <see langword="null" />.</value>
```